backend/presentation/views.py

```diff
@@ -17,6 +20,9 @@ class PresentationViewSet(ModelViewSet):
     serializer_class = OutputPresentationSerializer
     permission_classes = [IsAdminOrOwner]
     services = PresentationService()
+    filter_backends = (filters.DjangoFilterBackend,)
```

Comment:
Stay with one convention for iterative properties in views. 
```suggestion
    filter_backends = [filters.DjangoFilterBackend]
```

Reply:
You can also specify the default `DEFAULT_FILTER_BACKENDS` in the settings file.